### PR TITLE
docs: added Contributing blurb for a new package

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Documents the process for making a new package for future contributors.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@33.3.3-alpha.e80223.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605c8e3aed9ffd1c20662258--gamut-preview.netlify.app>https://605c8e3aed9ffd1c20662258--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605c8e3aed9ffd1c20662258>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

File: packages/styleguide/stories/Meta/Contributing.stories.mdx [hunk 1]
```diff
@@ -104,3 +102,18 @@ If you know your PR has breaking changes and/or is meant to be used in at least
...
+1. Copy and paste a similar existing package under `packages/`
+
+   > We recommend `markdown-overrides` for most basic packages.
+
+2. Modify its `package.json`'s `dependencies`, `description`, and `name` to match your new package
+3. Set its version to `0.1.0`
+4. Replace its `src/` directory with one only containing a `src/index.ts`
+5. Remove all content from its `CHANGELOG.md` after the _"See Conventional Commits ..."_ line
+6. From the root of the directory, run `yarn`, `yarn lerna bootstrap`, and `yarn build-all`
+
```
Comment (codecaaron) [inline packages/styleguide/stories/Meta/Contributing.stories.mdx:117]: maybe one to double check that you have `dist` in your .gitignore?

**Review by codecaaron — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@33.3.7-alpha.d2e47d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://605e1d6913195b00d13efce8--gamut-preview.netlify.app>https://605e1d6913195b00d13efce8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/605e1d6913195b00d13efce8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60633fa570838e0ea4d4b23a--gamut-preview.netlify.app>https://60633fa570838e0ea4d4b23a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60633fa570838e0ea4d4b23a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@33.4.1-alpha.badfab.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@33.6.4-alpha.fc7e10.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://606c97cb9961eb00d535d8e5--gamut-preview.netlify.app>https://606c97cb9961eb00d535d8e5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/606c97cb9961eb00d535d8e5>Deploy Logs</a></p>

**Review by aionate0812 — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@33.7.1-alpha.ad52da.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://606cab661f207d18cc5e0461--gamut-preview.netlify.app>https://606cab661f207d18cc5e0461--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/606cab661f207d18cc5e0461>Deploy Logs</a></p>

**Pull request merged**